homebrew/using_different_homebrew_formula_versions.md

```diff
@@ -0,0 +1,31 @@
+# Using different Homebrew formula versions
+
+Today, I was trying to run the executable from a gem and got the following error:
+
+```bash
+This installation of RMagick was configured with ImageMagick 6.8.9 but
+ImageMagick 6.9.0-3 is in use. (RuntimeError)
+```
```

Comment:
hmmm I think you're right about the bash syntax highlight not being good...but I still like it as a code block since I did first see it as an error message rather than a quote in a readme or something. will remove `bash`